modules/serverless-traffic-management-kn.adoc

```diff
@@ -0,0 +1,63 @@
+[id="serverless-traffic-management-kn_{context}"]
+= Traffic management using the Knative CLI
+
+You can use the `kn service update` command to split traffic between revisions of a service.
+
+.Example command
+[source,terminal]
+----
+$ kn service update <service_name> --traffic <revision>=<percent>
+----
+
+Where:
+
+* `<service_name>` is the name of the Knative service that you are configuring traffic routing for.
+* `<revision>` is the revision that you want to configure to receive a percentage of traffic. You can either specify the name of the revision, or a tag that you assigned to the revision by using the `--tag` flag.
+* `<percent>` is the percentage of traffic that you want to send to the specified revision.
+
+[id="serverless-traffic-management-kn-precedence"]
+== Multiple flags and order precedence
+
+All traffic-related flags can be specified using a single `kn service update` command. `kn` defines the precedence of these flags. The order of the flags specified when using the command is not taken into account.
+
+The precedence of the flags as they are evaluated by `kn` are:
+
+. `--untag`: All the referenced revisions with this flag are removed from the traffic block.
+. `--tag`: Revisions are tagged as specified in the traffic block.
+. `--traffic`: The referenced revisions are assigned a portion of the traffic split.
++
+[IMPORTANT]
+====
+The `--traffic` flag can be specified multiple times in one command, and is valid only if the sum of the `Percent` values in all flags totals 100.
+====
+
+You can add tags to revisions and then split traffic according to the tags you have set.
+
+[id="serverless-traffic-management-kn-example"]
+== Example
```

Comment:
Reminder about the ID attribute from preceding comment.

For the title, maybe "== Sample traffic management commands"  or "Example traffic management commands".